exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -138,6 +162,104 @@ public static TestKit forService(Class<? extends ServiceModule> serviceModule) {
     return serviceClass.cast(service);
   }
 
+  /**
+   * Creates a block with the given transaction. In-pool transactions will be ignored.
+   *
+   * @return created block
+   * @see <a href="https://exonum.com/doc/version/0.10/advanced/consensus/specification/#pool-of-unconfirmed-transactions">Pool of Unconfirmed Transactions</a>
+   */
+  public Block createBlockWithTransactions(TransactionMessage transaction,
+                                           TransactionMessage... transactions) {
+    return createBlockWithTransactions(asList(transaction, transactions));
+  }
+
+  /**
+   * Creates a block with the given transactions. In-pool transactions will be ignored.
+   *
+   * @return created block
+   * @see <a href="https://exonum.com/doc/version/0.10/advanced/consensus/specification/#pool-of-unconfirmed-transactions">Pool of Unconfirmed Transactions</a>
+   */
+  public Block createBlockWithTransactions(Iterable<TransactionMessage> transactions) {
+    List<TransactionMessage> messageList = Lists.newArrayList(transactions);
```

Comment:
this intermediate list can be replaced by `com.google.common.collect.Streams#stream(java.lang.Iterable<T>)`

Reply:
`Streams` are marked as `@Beta`, is it okay for us?

Reply:
No, for a library no.

Reply:
That's the reason we repackaged HashCode and friends.

Reply:
https://jira.bf.local/browse/ECR-1530